### PR TITLE
115 i18n component change comment

### PR DESCRIPTION
**Pull request by PeachyOmnivore**

### CODE REVIEW

File: src/components/comment/index.js [hunk 1]
```diff
@@ -4,11 +4,12 @@ import "../comment/comment.css";
 import ProfileCircle from "../profileCircle";
 import OptionsIcon from "../optionsIcon";
 import ProfileContextMenu from "../ProfileContextMenu";
+import { Trans } from 'react-i18next';
 
 const Comment = ({ user, content }) => { 
   const [triggerContextMenu, setTriggerContextMenu] = useState(false);
 
-  const fullName = user && user.firstName && user.lastName ? `${user.firstName} ${user.lastName}` : "Anonymous";
+  const fullName = user && user.firstName && user.lastName ? `${user.firstName} ${user.lastName}` : <Trans>"Anonymous"</Trans>;
```
Comment (PCapid3v) [inline src/components/comment/index.js:12]: Another great example!

**Review by PCapid3v — COMMENTED** [review]
Great!

File: src/components/comment/index.js [hunk 2]
```diff
@@ -4,11 +4,12 @@ import "../comment/comment.css";
...
   const initials = user && user.firstName && user.lastName ? `${user.firstName.charAt(0)}${user.lastName.charAt(0)}` : "A";
```
Comment (Uprising5034) [inline src/components/comment/index.js:13]: Should "A" be translated here as well?

Comment (PeachyOmnivore) [reply src/components/comment/index.js:13]: Think so as both words start with an A

**Review by Auenc — APPROVED** [review]

**Pull request merged**